components/library/http-parser/Makefile

```diff
@@ -0,0 +1,52 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Denys Rtveliashvili.  All rights reserved.
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		http-parser
+COMPONENT_VERSION=	2.6.2
+COMPONENT_FMRI=		library/http-parser
+COMPONENT_CLASSIFICATION=System/Libraries
+COMPONENT_PROJECT_URL=	https://github.com/nodejs/http-parser
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
+COMPONENT_ARCHIVE_HASH=	sha256:80fffc3b64ef6968cecdd4b299a96986007dff4bd12ae6c58cbcb506959b90ad
+COMPONENT_ARCHIVE_URL=	https://github.com/nodejs/http-parser/archive/v2.6.2.tar.gz
+COMPONENT_BUGDB=	$(COMPONENT_FMRI)
+COMPONENT_LICENSE=	MIT
+COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
+
+COMPONENT_SUMMARY=      A parser for HTTP messages written in C.
```

Comment:
Why is there a space between COMPONENT_SUMMARY and COMPONENT_LICENSE_FILE? Remove it please.